src/orion/core/io/database/mongodb.py

```diff
@@ -249,6 +264,15 @@ def write(self, collection_name, data, query=None):
         """
         dbcollection = self._db[collection_name]
 
+        if query is not None:
```

Comment:
Is this really using something specific to mongodb about ownership of documents or it's simply an additional uid that could also be used in the other database backends?

Reply:
It could be used for other database backends, ultimately the "database" backend is just mongodb 
and pickledb does not need the owner_id since there can only be one owner